apps/server/package.json

```diff
@@ -49,6 +49,7 @@
     "setdb": "shx cp ../../demo-db/db.json src/preloaded-db/db.json",
     "prebuild": "pnpm setdb",
     "build": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/index.cjs",
+    "build:docker": "pnpm prebuild && esbuild src/app.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/app.cjs && esbuild src/index.ts --log-level=error --platform=node --format=cjs --bundle --minify --outfile=dist/docker.cjs",
```

Comment:
Would it not be simpler to create the `docker.cjs` ourselves?
it is just a few lines of code that we manually maintain in electron anyway...

Reply:
how would we do this?

Reply:
we would simply create a file in a separate directory containing the code to start up the app.
you can see this code in the main.js inside electron

we would then use the dockerfile to move that docker.js into the built directory. we can have a look together if you would like

Reply:
lets take this on slack.
The short version is that the code in index.ts is very small and we could write it again, but there is nothing fundamentally wrong with what you've done, it sounds overcomplicated to me to get a build tool to write 10 lines of very straightforward code